### ISSUE TITLE
VERY slow save creation

### ISSUE THREAD

**Issue by cloudynetwork** [Lohikar/AuroraRecordGenerator#3]
On the latest version, I initially thought zero byte save files were created. However it seems to take some minutes for the save file to complete. This seems strange for such small files.

**Comment by Lohikar** [comment]
Do you have a third-party anti-virus running? I don't see why writing would be that slow unless something else was interfering with it.

**Comment by cloudynetwork** [comment]
Yes I do. I will do some more testing later on and see what is going on.

**Comment by cloudynetwork** [comment]
I've done a bit of testing and see no difference with AV on and off. I have noticed that there seems to be some coupling between the Save As, Save and Generate buttons. I notice that I don't actually get data saved to the specified save file unless I press Generate (or at least that's how it seems). This could be intentional functionality but would explain why the save file stays 0 bytes for 'some time'...
